### DISCUSSION TITLE
keymap configurator

### DISCUSSION THREAD

**Discussion by d3eOne** [nickcoutsos/keymap-editor#164] [General]
hi Nick, i have add the link on typeractive in order to tryout your keymap, 

I am fairly new in  zmk, i have try to add through the link and after installing the app, i only have as option to select "all repositpries" now the app is suspended, how to fix it, kindly please 

if easier for you my discord id is d3eOne #6161

![zmk keymap](https://github.com/nickcoutsos/keymap-editor/assets/127117071/79c0ec68-eacf-425c-9290-368589373330)

**Comment by nickcoutsos** [comment]
Hi @d3eOne, I've seen this before when people haven't yet created the repository. Fork the relevant zmk-config repository from typeractive and then you'll be able to select it for the keymap editor.

**Reply by d3eOne** [reply]
ok great thanks Nick, I will try it out again
